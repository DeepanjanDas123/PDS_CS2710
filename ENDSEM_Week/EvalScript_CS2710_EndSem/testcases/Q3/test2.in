GenerateTreeEnhanced
28
Q 1
e 3
O 4
j 6
W 7
B 8
L 10
X 11
9 18
n 19
N 21
Z 24
y 26
4 30
m 31
t 31
2 32
6 35
Y 38
D 39
J 39
i 39
E 43
x 43
C 47
K 49
0 50
f 50
Quit
