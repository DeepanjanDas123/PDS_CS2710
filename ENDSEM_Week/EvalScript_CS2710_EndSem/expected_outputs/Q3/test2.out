(((((9,n),Y),(D,J)),((i,(N,(L,X))),(E,x))),(((C,K),(0,f)),(((Z,y),(((j,W),(B,(O,(Q,e)))),4)),((m,t),(2,6)))));
